cloud_governance/policy/gcp/cost_billing_reports.py

```diff
@@ -0,0 +1,293 @@
+import json
+import os
+from datetime import datetime, timedelta
+from ast import  literal_eval
+
+from typeguard import typechecked
+
+from cloud_governance.common.clouds.gcp.google_account import GoogleAccount
+from cloud_governance.common.elasticsearch.elastic_upload import ElasticUpload
+from cloud_governance.common.google_drive.upload_to_gsheet import UploadToGsheet
+from cloud_governance.common.logger.logger_time_stamp import logger_time_stamp
+from cloud_governance.main.environment_variables import environment_variables
+from cloud_governance.common.logger.init_logger import logger
+
+
+class CostBillingReports:
+    """
+    This class is responsible for generation cost billing report for Budget, Actual, Forecast
+    """
+
+    DEFAULT_YEARS = 12
+    DEFAULT_ROUND_VALUE = 3
+    CLOUD_NAME = 'GCP'
+
+    def __init__(self):
+        self.__environment_variables_dict = environment_variables.environment_variables_dict
+        self.__database_name = self.__environment_variables_dict.get('GCP_DATABASE_NAME', '')
+        self.__table_name = self.__environment_variables_dict.get('GCP_DATABASE_TABLE_NAME', '')
+        self.__gcp_account = GoogleAccount()
+        self.__gsheet_id = self.__environment_variables_dict.get('SPREADSHEET_ID')
+        self.update_to_gsheet = UploadToGsheet()
+        self.elastic_upload = ElasticUpload()
+
+    @logger_time_stamp
+    def __next_twelve_months(self):
+        """
+        This method returns the next 12 month, year
+        :return:
+        """
+        months = 12
+        year = datetime.now().year
+        c_month = datetime.now().month + 1
```

Comment:
c_month, what is means  c_month ?

Reply:
c_month denotes - the current month. 
I changed it to next_month